benchmark/benchmain/main.go

```diff
@@ -143,7 +143,8 @@ var (
 		networkModeWAN:   latency.WAN,
 		networkLongHaul:  latency.Longhaul,
 	}
-	keepaliveTime    = 10 * time.Second // this is the minimum allowed
+	keepaliveTime    = 10 * time.Second       // This is the minimum allowed
+	keepaliveMinTime = 800 * time.Millisecond // This is 0.8*keepaliveTime
```

Comment:
`// This is false.`  :stuck_out_tongue: 

Did you mean to set keepaliveTime to 1s?

`Time` is hard-coded, or is it configurable with a flag?  If it is configurable, please compute `MinTime` from `Time` (where it is set in the enforcement policy, below).

Reply:
:man_facepalming:

Changed it to 8000millis. `Time` is hardcoded here. I guess the main point in the benchmarks to enable keepalives is to see the performance impact introduced by the code which maintains the `lastRead` time. So, the actual `time/timeout` values of the keepalive configurations don't matter as much.

Reply:
Makes sense, sounds reasonable.